packages/stack/src/views/Stack/CardSheet.tsx

```diff
@@ -36,6 +36,19 @@ export const CardSheet = React.forwardRef<CardSheetRef, Props>(
       const width = document.body.clientWidth;
       const height = document.body.clientHeight;
 
+      // Workaround for mobile Chrome, necessary when a navigation happens
+      // when the address bar has already collapsed, which resulted in an
+      // empty space at the bottom of the page (matching the height of the
+      // address bar). To fix this, it's necessary to update the height of
+      // the DOM with the current height of the window.
+      // See https://css-tricks.com/the-trick-to-viewport-units-on-mobile/
+      const vh = window.innerHeight * 0.01;
+      document.documentElement.style.setProperty('--vh', `${vh}px`);
+      document.body.setAttribute(
+        'style',
+        `height: calc(var(--vh, 1vh) * 100);`
+      );
+
       setFill(width === layout.width && height === layout.height);
```

Comment:
This thing which I'm still worried about is that in some apps, authors set the `body`'s style to `height: 100%`, in which case the collapsing won't happen. Alternatively, you can render the stack in a smaller part of the page. Then our fix is no longer valid and may break some things (better not to hack, if not needed). 


```suggestion
      const fillHeight = height === layout.height;
      if (fillHeight) {
        const vh = window.innerHeight * 0.01;
        document.documentElement.style.setProperty('--vh', `${vh}px`);
        document.body.setAttribute(
          'style',
          `height: calc(var(--vh, 1vh) * 100);`
        );
        workaroundApplied.current = true;
      } else { 
        if (workaroundApplied.current) {
          // revert the fix if it was set
          // e.g., document.documentElement.style.removeProperty('--vh');
        }
      }

      setFill(width === layout.width && height === layout.height);
```

Reply:
That makes sense, thank you 👍 I've applied the fix